linera-client/src/client_options.rs

```diff
@@ -194,8 +194,10 @@ impl ClientOptions {
         } else {
             cfg_if::cfg_if! {
                 if #[cfg(all(feature = "rocksdb", feature = "fs"))] {
+                    // The block_in_place is enabled since the client is run in multi-threaded
                     let storage_config = crate::storage::StorageConfig::RocksDb {
                         path: self.config_path()?.join("wallet.db"),
+                        block_in_place: true,
```

Comment:
Can you use the formula that queries the tokio runtime?

Reply:
Ok, done.